apps/admin-x-activitypub/src/components/layout/Onboarding/Step1.tsx

```diff
@@ -8,7 +8,7 @@ import {useBrowseUsers} from '@tryghost/admin-x-framework/api/users';
 import {useNavigate} from '@tryghost/admin-x-framework';
 
 const Step1: React.FC = () => {
-    const {data: account} = useAccountForUser('index');
+    const {data: account} = useAccountForUser('index', 'index');
```

Comment:
🤔 I mentioned this in the backend PR, but it might be nicer to assume the current user if this param isn't passed? That way we are not hardcoding `index` for the 2nd param?